### PR TITLE
comments please resolve

### PR DESCRIPTION
**Pull request by hagaimaya**

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// remove?
```
Comment (hagaimaya) [inline src/App.test.js:1]: please check if you need this file, if not please remove

File: src/index.css [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* remove? */
```
Comment (hagaimaya) [inline src/index.css:1]: check if you need this file, if not please remove

File: src/index.js [hunk 3]
```diff
@@ -2,15 +2,20 @@ import React from 'react';
...
 import reportWebVitals from './reportWebVitals';
 
+
+
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root') // check if you need this code
```
Comment (hagaimaya) [inline src/index.js:15]: check if you need this line.
learn how to explain this line

File: src/reportWebVitals.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+// remove file
```
Comment (hagaimaya) [inline src/reportWebVitals.js:1]: please check if you need this file

File: src/App.js [hunk 5]
```diff
@@ -2,59 +2,71 @@ import "./assets/styles.css";
 import Player from "./components/player";
 import Button from "./components/button";
 import Dice from "./components/dice";
+
 import { useState } from "react";
 
-function App() {
```
Comment (hagaimaya) [inline src/App.js:7]: change the way you write function in a consistent way

File: src/App.js [hunk 6]
```diff
@@ -2,59 +2,71 @@ import "./assets/styles.css";
...
   const [diceNumber, setDiceNumber] = useState(1);
   const [scorePlayer1, setScorePlayer1] = useState(0);
   const [scorePlayer2, setScorePlayer2] = useState(0);
-  const [currentPlayer1Score, setCurrentPlayer1Score] = useState(0);
-  const [currentPlayer2Score, setCurrentPlayer2Score] = useState(0);
+  const [currentScorePlayer1, setcurrentScorePlayer1] = useState(0);
+  const [currentScorePlayer2, setcurrentScorePlayer2] = useState(0);
+
+  // change name to showDice
   const [diceExist, setDiceExist] = useState(false);
```
Comment (hagaimaya) [inline src/App.js:16]: change name

File: src/App.js [hunk 7]
```diff
@@ -2,59 +2,71 @@ import "./assets/styles.css";
...
   const [player1Turn, setPlayer1Turn] = useState(true);
+  // should be player name
   const [winner, setWinner] = useState(false);
```
Comment (hagaimaya) [inline src/App.js:19]: change name

File: src/App.js [hunk 8]
```diff
@@ -2,59 +2,71 @@ import "./assets/styles.css";
...
 
+  // turn?
   const handleRollDice = () => {
```
Comment (hagaimaya) [inline src/App.js:22]: change to more meaningful name

File: src/App.js [hunk 9]
```diff
@@ -2,59 +2,71 @@ import "./assets/styles.css";
...
     if (!winner) {
       let randomNumber = Math.floor(Math.random() * 6) + 1;
       setDiceNumber(randomNumber);
       setDiceExist(true);
       if (player1Turn) {
+        // check how to use "usestate" for nested object
```
Comment (hagaimaya) [inline src/App.js:28]: read comments

File: src/App.js [hunk 10]
```diff
@@ -2,59 +2,71 @@ import "./assets/styles.css";
...
       }
     }
   };
 
+  // what is hold
   const handleHoldClick = () => {
     if (!winner) {
       if (player1Turn) {
-        let current = scorePlayer1 + currentPlayer1Score;
+        // make a function
```
Comment (hagaimaya) [inline src/App.js:55]: duplicate code make a function

File: src/App.js [hunk 11]
```diff
@@ -63,16 +75,25 @@ function App() {
     }
   };
 
+  // reset game
```
Comment (hagaimaya) [inline src/App.js:78]: rename

File: src/App.js [hunk 12]
```diff
@@ -99,16 +120,17 @@ function App() {
         playerSide="left-player"
         playerName="PLAYER 1"
         score={scorePlayer1}
-        roll={currentPlayer1Score}
-        currentPlaying={player1Turn ? "current-playing" : ""}
+        roll={currentScorePlayer1}
+        currentPlaying={player1Turn ? msg.currentPlaying : msg.notPlaying}
```
Comment (hagaimaya) [inline src/App.js:124]: change hardcoded strings to a variable

**Review by hagaimaya — COMMENTED** [review]
@rotemshaked  please fix the comments

**Review by rotemshaked — APPROVED** [review]

**Pull request merged**